components/layout/MainMenu.tsx

```diff
@@ -1,81 +1,187 @@
-import { Suspense } from "react";
-import { LuChevronDown } from "react-icons/lu";
+"use client";
```

Comment:
This directive turns the entire page into a Client Component. It should only be used in the relevant section and wrapped with Suspense.